### ISSUE TITLE
Story filters activity

### ISSUE THREAD

**Issue by MarcAnt01** [huynhsontung/Indirect#22]
It's impossible to see when I send or receive filters in directs

**Comment by huynhsontung** [comment]
What do you mean by filter? I don't think I have seen this item type before.

**Comment by MarcAnt01** [comment]
> What do you mean by filter? I don't think I have seen this item type before.

You are right, they aren't called filters, but EFFECTS (sorry got confused with Snapchat)

**Comment by huynhsontung** [comment]
I still don't know what those are. Can you provide an example?

**Comment by MarcAnt01** [comment]
![image](https://user-images.githubusercontent.com/43090263/77409615-d8173b80-6db9-11ea-85ee-eb669754d8b4.png)

**Comment by huynhsontung** [comment]
Isn't it already embedded to video/picture? Is there a benefit to know specific type of effect?

**Comment by MarcAnt01** [comment]
> Isn't it already embedded to video/picture? Is there a benefit to know what type of effect used in a photo/video?

You can send a specific effect to a user, to make him try it

**Comment by huynhsontung** [comment]
That's fair. It should already be returned along with the story object. Showing this should be relatively simple I think 👍
